### DISCUSSION TITLE
POV: You're using GraphiQL

### DISCUSSION THREAD

**Discussion by joaohcopetti** [graphql/graphiql#3201] [General]
![alt text](https://i.imgur.com/3QfCC0o.png)

_Edited by @thomasheyenbrock: The original message was rude and insulting, thus removed by the maintainers. It was hinting at a bug that we already track in an issue (see the comment below)._

**Comment by thomasheyenbrock** [comment]
Hi @JoaoHamerski, we already have an issue that is tracking this bug: https://github.com/graphql/graphiql/issues/3107
